doc/sphinx/developer_manual/libraries/utilities/utilities_library.rst

```diff
@@ -11,7 +11,7 @@ Structure
 =========
 The header files of the utilities library are roughly organised as depicted below.
 
-.. figure:: img/layout.png
+.. figure:: ../../../_static/utilities/layout.png
```

Comment:
Wouldn't it be better to store this layout.png in the same folder or a subfolder (as it was before) to keep filepaths short and keep things together that belong together? If there is a considerate amount op reuse of such files then I understand why you would want these files in a separate place. This comment is applicable to more files.

Reply:
You might be able to solve this using /_static/utilities/layout.png, where the first / indicates that the path is relative to the conf.py according to the answer given here: https://stackoverflow.com/questions/8475865/in-python-sphinx-how-to-link-to-a-file-in-static-directory

Reply:
The main reason for moving the images was to have a separation of images and text, much like ideally we separate source and build files. I shortened the paths as suggested by @mlaveaux in 71dd4fee3ab612ef93171b0f338e683acf853464 and hope that resolves this comment.